### DISCUSSION TITLE
Windows installation of filesystem

### DISCUSSION THREAD

**Discussion by Nimr1** [modelcontextprotocol/.github#63] [Q&A]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [X] Server Implementation
- [ ] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

Hi, I have tried the recommended config file however it still doesnt pick up the filesystem. 

Could someone give a bit more context on why and what I need to put into the claude_desktop_config.json file

**Comment by Jacck** [comment]
You need node,js installed on Windows.
Run command: npm install @modelcontextprotocol/server-filesystem
Copy path: "C:\Users\user\AppData\Roaming\npm\node_modules\@modelcontextprotocol\server-filesystem\dist\index.js
Add to claude_desktop_config the following json:
{
  "filesystem": {
    "command": "node",
    "args": [
      "C:\\Users\\user\\AppData\\Roaming\\npm\\node_modules\\@modelcontextprotocol\\server-filesystem\\dist\\index.js",
      "/Users/user/Desktop",
      "C:\\Users\\user\\rest_of_the_path_to_granted_directory"
    ]
  }
}

**Reply by Nimr1** [reply]
What do i put in at "rest_of_the_path_to_granted_directory"

**Reply by Jacck** [reply]
the directory that you grant to Claude desktop
